activities/Falabracman.activity/activity/activity.info

```diff
@@ -1,5 +1,5 @@
 [Activity]
-name =  Activity
+name =  Falabracman Activity
```

Comment:
Should be "Falabracman" only else the menu is incorrect:

<img width="317" alt="image" src="https://user-images.githubusercontent.com/981841/73767322-9da8ff00-4777-11ea-8b07-90fbe9417fb4.png">